Failed to generate javadoc

### Area

storage

### Describe the bug

Keycloak release no longer work due to failure in generating JavaDoc for model/map-hot-rod. This is currently blocking the ability to do a release, including Nightly releases.

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:javadoc (aggregate-javadoc) on project keycloak-api-docs-dist: An error has occurred in Javadoc report generation: 
Error:  Exit code: 1 - /home/runner/work/keycloak-rel/keycloak-rel/model/map-hot-rod/target/generated-sources/annotations/org/keycloak/models/map/storage/hotRod/user/HotRodUserEntity$___Marshaller_1ca86473835481afa775515960874a2887b3c4fa81ee6cec155743567fe9b331.java:15: error: cannot find symbol
Error:  @javax.annotation.processing.Generated(
Error:                              ^
Error:    symbol:   class Generated
Error:    location: package javax.annotation.processing
Error:  /home/runner/work/keycloak-rel/keycloak-rel/model/map-hot-rod/target/generated-sources/annotations/org/keycloak/models/map/storage/hotRod/user/HotRodUserConsentEntity$___Marshaller_94f0c89a18d3b5525f6b3a9248b79b536f16a91cbd059c3be592bfb1d1394e5d.java:15: error: cannot find symbol
Error:  @javax.annotation.processing.Generated(
Error:                              ^
Error:    symbol:   class Generated
Error:    location: package javax.annotation.processing
Error:  /home/runner/work/keycloak-rel/keycloak-rel/model/map-hot-rod/target/generated-sources/annotations/org/keycloak/models/map/storage/hotRod/user/HotRodUserCredentialEntity$___Marshaller_20cea60b3d22077ccb7bf3e15c5e75dedab46990f0020c7ff267f96f8cf00bc5.java:15: error: cannot find symbol
Error:  @javax.annotation.processing.Generated(
Error:                              ^
Error:    symbol:   class Generated
Error:    location: package javax.annotation.processing
Error:  /home/runner/work/keycloak-rel/keycloak-rel/model/map-hot-rod/target/generated-sources/annotations/org/keycloak/models/map/storage/hotRod/user/HotRodUserFederatedIdentityEntity$___Marshaller_1e1ce2d49b32d4e6f3d48de84699ed86a17d11c291442dddfc60b2577f1862a3.java:15: error: cannot find symbol
Error:  @javax.annotation.processing.Generated(
Error:                              ^
Error:    symbol:   class Generated
Error:    location: package javax.annotation.processing
Error:  /home/runner/work/keycloak-rel/keycloak-rel/model/map-hot-rod/target/generated-sources/annotations/org/keycloak/models/map/storage/hotRod/user/HotRodUserEntitySchemaImpl.java:12: error: cannot find symbol
Error:  @javax.annotation.processing.Generated(
```

### Version

main

### Expected behavior

Release is successful

### Actual behavior

Release fails

### How to Reproduce?

Run `mvn clean install -Pjboss-release`

### Anything else?

Failure on nightly release: https://github.com/keycloak-rel/keycloak-rel/actions/runs/3493389086

--- a/distribution/api-docs-dist/pom.xml
+++ b/distribution/api-docs-dist/pom.xml
@@ -31,6 +31,7 @@
 
     <properties>
         <javadoc.branding>Keycloak ${project.version}</javadoc.branding>
+        <maven.compiler.release>11</maven.compiler.release>
     </properties>
 
     <dependencies>

--- a/federation/sssd/src/main/java/org/jvnet/libpam/PAM.java
+++ b/federation/sssd/src/main/java/org/jvnet/libpam/PAM.java
@@ -78,7 +78,7 @@ public class PAM {
      */
     public PAM(String serviceName) throws PAMException {
         pam_conv conv = new pam_conv(new PamCallback() {
-            public int callback(int num_msg, Pointer msg, Pointer resp, Pointer _) {
+            public int callback(int num_msg, Pointer msg, Pointer resp, Pointer _ptr) {
                 LOGGER.debug("pam_conv num_msg=" + num_msg);
                 if (factors == null)
                     return PAM_CONV_ERR;

--- a/federation/sssd/src/main/java/org/jvnet/libpam/impl/PAMLibrary.java
+++ b/federation/sssd/src/main/java/org/jvnet/libpam/impl/PAMLibrary.java
@@ -116,18 +116,18 @@ public interface PAMLibrary extends Library {
              * resp and its member string both needs to be allocated by malloc,
              * to be freed by the caller.
              */
-            int callback(int num_msg, Pointer msg, Pointer resp, Pointer _);
+            int callback(int num_msg, Pointer msg, Pointer resp, Pointer _ptr);
         }
 
         public PamCallback conv;
-        public Pointer _;
+        public Pointer _ptr;
 
         public pam_conv(PamCallback conv) {
             this.conv = conv;
         }
 
         protected List getFieldOrder() {
-            return Arrays.asList("conv", "_");
+            return Arrays.asList("conv", "_ptr");
         }
     }
 

